Samples/MediaPlayerCS/MainPage.xaml.cs

```diff
@@ -136,12 +136,12 @@ private void OnMediaFailed(object _1, ExceptionRoutedEventArgs args)
 		}
 
 		private async void OnError(string errMsg)
-		{
-			// Display error message
+		{
```

Comment:
same here. not sure what changed.

Reply:
It looks like the file had mixed line endings (CRLF vs LF) and VS Code auto-formatted it for consistency.